### PR TITLE
pythonPackages.backoff: init at 1.10.0

### PR DESCRIPTION
**Pull request by chkno**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested a program that uses this library
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
    0 → 110532768
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/backoff/default.nix [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "backoff";
+  version = "1.10.0";
+
+  src = fetchPypi {
```
Comment (geistesk) [inline pkgs/development/python-modules/backoff/default.nix:7]: The source on Pypi does not include tests. Perhaps you should switch the source to the GitHub repository.

> Ran 0 tests in 0.000s

**Review by oxzi — CHANGES_REQUESTED** [review]
Thanks for your PR.

Would you please change the `src` to include tests or otherwise explicitly disable the tests and try `pythonImportsCheck`.

File: pkgs/development/python-modules/backoff/default.nix (hunk 1 above)
Comment (chkno) [reply pkgs/development/python-modules/backoff/default.nix]: Thanks!  Done: It now says `58 passed` as it builds.

**Review by oxzi — APPROVED** [review]
LGTM!

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 93377 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>2 packages built:</summary><br>- python37Packages.backoff<br>- python38Packages.backoff</details>

**Pull request merged**